File: content/javascript/basic/05-objects/code.js
const person = {
    name: "Alice",
    age: 30,
    city: "Madrid",
};

console.log(`Name: ${person.name}`);
console.log(`Age: ${person.age}`);

person.email = "alice@example.com";

const { name, city } = person;
console.log(`${name} lives in ${city}`);

const keys = Object.keys(person);
console.log(`Keys: ${keys}`);

File: content/python/basic/05-dictionaries/code.py
person = {
    "name": "Alice",
    "age": 30,
    "city": "Madrid"
}

print(f"Name: {person['name']}")
print(f"Age: {person['age']}")

person["email"] = "alice@example.com"

for key, value in person.items():
    print(f"  {key}: {value}")
